canonical/ubuntu.com
# WD-16445 Pick product listings with the right financial quarter

## Done

- Pick product listings with the right financial quarter
- **How it should work**: there are product listings from the backend. When a user selects an offer, only the product listings matching the offer's exclusion group should be available in the shop. For example, product listings with different exclusion groups may exist for the same products such as `2024-Q1` or `2024-quarter-2` (names can be any string). If an offer's exclusion group is `2024-Q4`, only product listings with `2024-Q4` should be available.

## QA

- Check out this feature branch or [demo](https://ubuntu-com-14554.demos.haus/)
- Go to /pro/distributor
- Select offer
- Check adding/removing subscriptions in the step 4 on the shop are working properly and check the summary at the bottom changes correctly
- Check everything working fine on the shop page
- Click "Proceed to checkout" to go to the checkout page
- Update info if necessary 
- Check making a purchase works as expected. 

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-16445
